test/integration/spec/preflight.js

```diff
@@ -1,15 +1,128 @@
-/* eslint-disable no-console, no-undefined */
+/* eslint-disable no-undefined */
 'use strict';
 
 const assert = require('assert');
-const { runPreflight } = require('../../../lib');
+// const testPreflight = require('../../../lib/preflight/preflight');
+const runPreflight = require('../../../es5/index').runPreflight;
```

Comment:
We also probably don't need to reference es5 here?

Reply:
I think we still need this, as we are not compiling test `js` code. it takes dependency on generated (es5) lib code.

Reply:
Ok got it.